upgrade/upgrade-scripts/common.sh

```diff
@@ -268,6 +272,7 @@ function source_version_information() {
 	[[ -n "$MINIMUM_VERSION" ]] || die "MINIMUM_VERSION is empty"
 	[[ -n "$MINIMUM_REBOOT_OPTIONAL_VERSION" ]] ||
 		die "MINIMUM_REBOOT_OPTIONAL_VERSION is empty"
+	IMAGE_VERSION="$VERSION"
```

Comment:
If we remove the `get_image_version` function, I think we can remove this as well.. I don't think we need the `IMAGE_VERSION` variable, do we?

Reply:
I was attempting to minimize changes to avoid introducing simple bugs while working on bigger issues. However now that these bigger issues are not as much a concern, I certainly agree that cleanup work makes some sense.